ros2controlcli/ros2controlcli/verb/set_controller_state.py

```diff
@@ -81,7 +81,7 @@ def main(self, *, args):
 
                 else:
                     return (
-                        f'cannot put {matched_controller.name} in "inactive" state'
+                        f'cannot put {matched_controller.name} in "inactive" state '
```

Comment:
This looks a bit better to me :)
```suggestion
                        f"cannot put {matched_controller.name} in 'inactive' state "
```